configs/config.protectli_cml_vp4670

```diff
@@ -25,21 +25,30 @@ CONFIG_TPM_MEASURED_BOOT=y
 CONFIG_BOOTMEDIA_LOCK_CONTROLLER=y
 CONFIG_BOOTMEDIA_LOCK_WPRO_VBOOT_RO=y
 CONFIG_BOOTMEDIA_LOCK_IN_VERSTAGE=y
+CONFIG_BOOTMEDIA_SMM_BWP=y
 CONFIG_DEFAULT_CONSOLE_LOGLEVEL_0=y
 # CONFIG_CONSOLE_USE_LOGLEVEL_PREFIX is not set
 # CONFIG_CONSOLE_USE_ANSI_ESCAPES is not set
 CONFIG_POST_IO_PORT=0x80
 CONFIG_PAYLOAD_EDK2=y
 CONFIG_EDK2_REPOSITORY="https://github.com/Dasharo/edk2"
-CONFIG_EDK2_TAG_OR_REV="origin/protectli_lan_rom_removal"
+CONFIG_EDK2_TAG_OR_REV="origin/unified_ipxe"
```

Comment:
Yes it should. Not sure what to do with that since merging https://github.com/Dasharo/edk2/pull/34 may change the commit hashes.

Reply:
@miczyg1 The linked MR was merged 4 days ago. Do you mean another one?
Mayb we may also use tags here?

Reply:
We have multiple PRs hanging on EDK2. Each merge will changes hashes. Do you suggest tags for every PR? If anything, I would update the edk2 tag or rev Kconfig value right before release to hook onto the code we want to be included in the release.

Secondly, edk2/dasharo branch has a few features that would require integration on coreboot side here as well. I didn;t do it because that was not in scope of my task. If it really bothers you that much, I will not include any changes to edk2 revisions in miniconfigs anymore...

Reply:
> We have multiple PRs hanging on EDK2

Do we? https://github.com/Dasharo/edk2/pulls

> Do you suggest tags for every PR

I do not suggest anything. Simply throwing ideas, that maybe using
tags for edk2 Dasharo releases (insterad of raw commit hash) might be worh
considering, or more convenient.

> I would update the edk2 tag or rev Kconfig value right before release to hook onto the code we want to be included in the release.

I do not think anyone has problem with that. Unless you explaining it from the beginning. You said you want to hold off until https://github.com/Dasharo/edk2/pull/34 is merged. It was (so based on your comment, one could assume we can throw in a revision now it is merged). I asked what's next and this happend out of nowhere.

> Secondly, edk2/dasharo branch has a few features that would require integration on coreboot side here as well.

Which ones?

> I didn;t do it because that was not in scope of my task.

Why is it not explained in the MR description or comments, what's missing? From your comments I have impression that this MR will not render usable build. Is it true?

>  If it really bothers you that much, I will not include any changes to edk2 revisions in miniconfigs anymore...

I will simply better not comment that.

Reply:
Set fixed revision.